python/frequencies_converter/models/allocation_table.py

```diff
@@ -86,6 +88,19 @@ def to_data_file_directory(self, directory: str) -> None:
             year=self.year,
         )
 
+    def to_protobuf(self) -> frequencies_pb2.FrequencyAllocations:
```

Comment:
there has to be a simpler way to do this but given im not too familiar i have written it this way so it's runnable.

Will look into simplifying this later. maybe we can convert this to dict and proto automatically with built in tools?